### ISSUE TITLE
SA1513 (ClosingBraceMustBeFollowedByBlankLine) not fired inside lambda

### ISSUE THREAD

**Issue by whut** [DotNetAnalyzers/StyleCopAnalyzers#2062]
SA1513 (ClosingBraceMustBeFollowedByBlankLine) not fired inside lambda. Tested under StyleCopAnalyzers 1.0.0.

**Example**

SA1513 should be found on the closing parenthesis line before the second 'if'

``` csharp
Task.Run(() =>
{
    if (true)
    {
        Console.WriteLine();
    }
    if (true)
    {
        Console.WriteLine();
    }
});
```

**Comment by pdelvo** [comment]
Hey @whut,
Thanks for reporting this. I agree that a diagnostic should be reported between  the two if statements.

**Comment by pdelvo** [comment]
It looks like this only happens inside visual studio. It does not reproduce in test cases, on the command line compiler and not inside the visual studio test hive when debugging

**Comment by sharwell** [comment]
In release 1.0.0, SA0000 is disabled by default. In the current master branch for 1.1.0, it is enabled by default with Hidden severity, which means it is enabled for files open in the editor.

1. Can you reproduce it in any case with 1.0.0 when SA0000 is (manually) enabled with Hidden severity?
2. Can you reproduce it in any case with 1.1.0?

**Comment by bjornhellander** [comment]
In current master, this reproduces just fine. Line 242 in [SA1513ClosingBraceMustBeFollowedByBlankLine.cs](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/bd617d04c67d8fb778b2a2acf563f1510723d924/StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1513ClosingBraceMustBeFollowedByBlankLine.cs) explicitly ignores all braces inside argument lists.

I would expect the closing brace of the lambda to be ignored by this rule, but not the braces **inside** the lambda. Any other thoughts?
